format/view-spec.md

```diff
@@ -0,0 +1,264 @@
+
+| Required/Optional | Field Name | Description |
+|-------------------|------------|-------------|
+| Required | timestamp-ms | Tthe timestamp hen the referenced version was made the current version |
+| Required | version-id | Version id of the view  |
+
+#### Summary
+
+Field “summary” is a string map with the following keys. Only `operation` is required. Engines may store additional key-value pairs in this map.
+
+| Required/Optional | Key | Value |
+|-------------------|-----|-------|
+| Required | operation | A string value indicating the view operation that caused this metadata to be created. Allowed values are “create” and “replace”. |
+| Optional | engine-version | A string value indicating the version of the engine that performed the operation |
+
+#### Representations
+
+Each representation is stored as an object with only one common field "type".
+The rest of the fields are interpreted based on the type.
+There is only one type of representation defined in the spec.
+
+##### Original View Definition in SQL
+
+This type of representation stores the original view definition in SQL and its SQL dialect.
+
+| Required/Optional | Field Name | Description |
+|-------------------|------------|-------------|
+| Required | type | A string indicating the type of representation. It is set to "sql" for this type. |
+| Required | sql | A string representing the original view definition in SQL |
+| Required | dialect | A string specifying the dialect of the ‘sql’ field. It can be used by the engines to detect the SQL dialect. |
+| Optional | schema-id | ID of the view's schema when the version was created |
+| Optional | default-catalog | A string specifying the catalog to use when the table or view references in the view definition do not contain an explicit catalog. |
+| Optional | default-namespace | The namespace to use when the table or view references in the view definition do not contain an explicit namespace. Since the namespace may contain multiple parts, it is serialized as a list of strings. |
+| Optional | field-aliases | A list of strings of field aliases optionally specified in the create view statement. The list should have the same length as the schema's top level fields. See the example below. |
+| Optional | field-docs | A list of strings of field comments optionally specified in the create view statement. The list should have the same length as the schema's top level fields. See the example below. |
+
+For `CREATE VIEW v (alias_name COMMENT 'docs', alias_name2, ...) AS SELECT col1, col2, ...`,
+the field aliases are 'alias_name', 'alias_name2', and etc., and the field docs are 'docs', null, and etc.
+
+## Appendix A: An Example
```

Comment:
Rather than an example for interpretation, could we do the same thing as the table spec and have a JSON serialization section with good examples at a field level?

You could put a real SQL `CREATE VIEW` statement at the top and base the example on that. I think that would be more helpful because the description would be more specific than big pasted blobs of JSON with comments.